students/jjakub/session02/grid03.py

```diff
@@ -0,0 +1,12 @@
+def print_grid2(x,y):
+
+    plus = (("+" + "-"*y) * x) + "+"
+    pipe = (("|" + " "*y) * x) + "|"
+
+    for row in range((y * x) + x + 1):
+        if row%(y + 1) == 0:
+            print(plus)
+        else:
+            print(pipe)
```

Comment:
Interesting, seemingly smart logic, and I think it's going to work (purely from reading the code), but I'll let TA do the more in-depth grading.

Reply:
Indeed, a very clever and elegant solution. (and correct) Also, it is relatively good readable.